lightning/src/ln/splicing_tests.rs

```diff
@@ -157,3 +157,61 @@ fn test_v1_splice_in() {
 		acceptor_node.node.get_our_node_id()
 	);
 }
+
+#[test]
+fn test_v1_splice_in_negative_inufficient_inputs() {
+	let chanmon_cfgs = create_chanmon_cfgs(2);
+	let node_cfgs = create_node_cfgs(2, &chanmon_cfgs);
+	let node_chanmgrs = create_node_chanmgrs(2, &node_cfgs, &[None, None]);
+	let nodes = create_network(2, &node_cfgs, &node_chanmgrs);
+
+	// Initiator and Acceptor nodes
+	let initiator_node_index = 0;
+	let acceptor_node_index = 1;
+	let initiator_node = &nodes[initiator_node_index];
+	let acceptor_node = &nodes[acceptor_node_index];
+
+	// Instantiate channel parameters where we push the maximum msats given our funding satoshis
+	let channel_value_sat = 100_000; // same as funding satoshis
+	let push_msat = 0;
+
+	let (_, _, channel_id, _) = create_announced_chan_between_nodes_with_value(
+		&nodes,
+		initiator_node_index,
+		acceptor_node_index,
+		channel_value_sat,
+		push_msat,
+	);
+
+	// ==== Channel is now ready for normal operation
+
+	// === Start of Splicing
+	println!("Start of Splicing ..., channel_id {}", channel_id);
+
+	// Amount being added to the channel through the splice-in
+	let splice_in_sats: u64 = 20_000;
+
+	// Create additional inputs, but insufficient
+	let extra_splice_funding_input_sats = 19_900;
+	assert!(extra_splice_funding_input_sats <= splice_in_sats);
+	let funding_inputs = create_dual_funding_utxos_with_prev_txs(
+		&initiator_node,
+		&[extra_splice_funding_input_sats],
+	);
+	// Initiate splice-in, with insufficient input contribution
+	let res = initiator_node.node.splice_channel(
+		&channel_id,
+		&acceptor_node.node.get_our_node_id(),
+		splice_in_sats as i64,
+		funding_inputs,
+		1024, // funding_feerate_per_kw,
+		None, // locktime
+	);
+	assert!(res.is_err());
+	match res.as_ref().err().unwrap() {
+		crate::util::errors::APIError::APIMisuseError { err } => {
+			assert!(err.contains("nsufficient"))
```

Comment:
Could you use "Insufficient  inputs for splicing"?

Reply:
Yes, more clear this way. The more details are in the test, the more likely it will break (with change), but that should not happen too often.